components/Alert/Alert.tsx

```diff
@@ -5,7 +5,7 @@ import * as React from 'react';
 import { cva, type VariantProps } from 'class-variance-authority';
 
 const alertVariants = cva(
-  'relative w-full rounded-lg border p-4 [&>svg~*]:pl-7 [&>svg+div]:translate-y-[-3px] [&>svg]:absolute [&>svg]:left-4 [&>svg]:top-4 [&>svg]:text-foreground',
+  'relative w-full max-w-[25rem] rounded-lg border p-4 [&>svg~*]:pl-7 [&>svg+div]:translate-y-[-3px] [&>svg]:absolute [&>svg]:left-4 [&>svg]:top-4 [&>svg]:text-foreground',
```

Comment:
```suggestion
  'relative w-full md:max-w-[25rem] rounded-lg border p-4 [&>svg~*]:pl-7 [&>svg+div]:translate-y-[-3px] [&>svg]:absolute [&>svg]:left-4 [&>svg]:top-4 [&>svg]:text-foreground',
```

The way you have this written currently is that first it'll look to apply `w-full` to the `Alert`, but then say "wait a minute, it should be `max-w-[25rem]`. By adding the `md:` in front of `max-w-[25rem]` we are able to control at what breakpoint it applies that max-width.

It might not be a problem now, but we should be as specific as possible and truly control the CSS.